meta/main.yml

```diff
@@ -38,25 +59,3 @@ galaxy_info:
     #     - focal
     #     - jammy
   role_name: mongo
```

Comment:
Please remove Fedora 35, even though it is commented out.  We don't want it to be mistakenly brought back from the dead.

Reply:
Resolved by commit a00dcb8.